addons/website_version/models/experiment.py

```diff
@@ -0,0 +1,300 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, timedelta
+import logging
+from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from openerp.exceptions import Warning
+from openerp import models, fields, exceptions, SUPERUSER_ID, api
+from openerp.tools.translate import _
+import simplejson
+
+_logger = logging.getLogger(__name__)
+
+
+class Experiment_version(models.Model):
+
+    _name = "website_version.experiment_version"
+
+    version_id = fields.Many2one('website_version.version', string="version_id", required=True, ondelete='cascade')
+    experiment_id = fields.Many2one('website_version.experiment', string="Experiment_id", required=True, ondelete='cascade')
+    frequency = fields.Selection([('10', 'Less'), ('50', 'Medium'), ('80', 'More')], string='Frequency', default='50')
+    google_index = fields.Integer(string='Google index')
+
+
+#This model allow to define the Google analytics goals in a experiment
+
+
+class Goals(models.Model):
+
+    _name = "website_version.goals"
+
+    name = fields.Char(string="Name", required=True)
+    google_ref = fields.Char(string="Reference Google", required=True)
+
+
+class Experiment(models.Model):
+
+    _name = "website_version.experiment"
+    _inherit = ['mail.thread']
+    _order = 'sequence'
+
+    @api.multi
+    def _check_view(self):
```

Comment:
I don't understand why ('experiment_id', 'not in', running.ids) and ('experiment_id.state', '=', 'running') because it seems that one condition is the negation of the other one.